base/linalg/bidiag.jl

```diff
@@ -104,6 +103,8 @@ function convert{T}(::Type{Matrix{T}}, A::Bidiagonal)
     return B
 end
 convert{T}(::Type{Matrix}, A::Bidiagonal{T}) = convert(Matrix{T}, A)
+convert(::Type{Array}, A::Bidiagonal) = convert(Matrix, A)
+full(A::Bidiagonal) = convert(Array, A)
```

Comment:
> Should we deprecate full while you're at it?

Please see the [response below](https://github.com/JuliaLang/julia/pull/17066#issuecomment-228113687). Thanks!